.github/workflows/pre-commit-autoupdate.yml

```
@@ -0,0 +1,74 @@
+name: Pre-commit auto-update
+
+on:
```

```suggestion
on: # yamllint disable-line rule:truthy
```

this should appease the failing yaml lint.